src/components/Text/Text.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { px } from '../../utils';
+import classNames from 'classnames';
```

Comment:
Can we standardize the import for this? I think we have more `classnames` for this package import, which I prefer because it matches the package and does not get confused with the attribute.